src/convnets/builders/resnet.jl

```diff
@@ -0,0 +1,9 @@
+function resnetbuilder(img_dims, stem, get_layers, block_repeats::AbstractVector{<:Integer},
```

Comment:
It's a small thing, but I wonder if these functions which actually return a `Chain` instead of a builder function should be named with a leading imperative. e.g.
```suggestion
function build_resnet(img_dims, stem, get_layers, block_repeats::AbstractVector{<:Integer},
```
Then there is a separation between the functions which give you back something to call later, and the ones which give you a model immediately.

Reply:
Makes sense. Names _are_ so important here because they really should try and convey the point of the function even before the user starts reading the docs